.gitignore

```diff
@@ -48,3 +48,5 @@ jspm_packages
 # Lockfiles
 yarn.lock
 package-lock.json
+
+.env
```

Comment:
Would be good if we would ignore `.env.local` instead, I think, so that `.env` can be checked in and used for defaults?